twython/endpoints.py

```diff
@@ -199,28 +199,33 @@ def upload_video(self, media, media_type, size=None):
             'command': 'FINALIZE',
             'media_id': media_id
         }
-        response_finalize = self.post(upload_url, params=params)
 
-        # Stage 4: STATUS call if still processing
-        params = {
-            'command': 'STATUS',
-            'media_id': media_id
-        }
-        
-        processing_state = response_finalize['processing_info'].get('state', None)
+        # Only get the status if explicity asked to 
+        # Default to False
+        if check_progress == True:
+
+            response_finalize = self.post(upload_url, params=params)
```

Comment:
Can you make this:

```python
response = self.post(upload_url, params=params)
```

outside of this `if check_progress:`

and use `response` everywhere you use `response_finalize`

and outside of the check, you can just `return response` then, does that make sense or should I elaborate more?